flipperdevices/flipperzero-firmware
# SD Driver: reinit sd card on error

# What's new

- Storage: code refactoring
- Storage: try to remount SD on IO error

# Verification 

- Check storage working in update and normal modes
- Check storage correct work on SD Card failures (errors can be injected by short circuiting ext 3v3 while IO)

# Checklist (For Reviewer)
